lib/spack/spack/test/cmd/install.py

```diff
@@ -141,7 +141,7 @@ def test_package_output(capsys, install_mockery, mock_fetch):
     # when nested AND in pytest
     spec = spack.concretize.concretize_one("printing-package")
     pkg = spec.package
-    PackageInstaller([pkg], explicit=True, verbose=True).install()
+    PackageInstaller([pkg], explicit=True, verbose=True, tests=sys.platform != "win32").install()
```

Comment:
We normally skip tests that don't work on windows. So the preferred solution is to mark it for skipping with a reason using `@pytest.mark.skipif()`. See `test_concurrent_packages_set_in_config` in the same file for an example.

Reply:
🤔 I should've looked more closely at the test. I'm fine with keeping it as is under the circumstances. *Thanks* for pointing that out to me.

Reply:
Ah. You were faster than my response.